README.md

```diff
@@ -37,42 +53,32 @@
 
 ## Introduction
 
-Teleport is the easiest, most secure way to access all your infrastructure.
-Teleport is an identity-aware, multi-protocol access proxy which understands
-SSH, HTTPS, RDP, Kubernetes API, MySQL, MongoDB and PostgreSQL wire protocols.
+Teleport includes an identity-aware access proxy, a CA that issues short-lived certificates, and an access control and tunneling system to access resources behind the firewall.
```

Comment:
The access control system and tunneling system are separate things. This makes them sound the same. (You can use access control without tunneling)